src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
@JoelKatz How is the code you linked to any less clear than the code below, which reduces the likelihood of conflicting with the stdlib?
```
    state_ = State::open;
    closeTime_ = closeTime;
    prevLedgerHash_ = prevLCLHash;
    previousLedger_ = prevLedger;
    ourPosition_.reset();
    consensusFail_ = false;
    currentMSeconds_ = 0ms;
    closePercent_ = 0;
    haveCloseTimeConsensus_ = false;
    consensusStartTime_ = std::chrono::steady_clock::now();
    previousProposers_ = previousProposers;
    previousMSeconds_ = previousConvergeTime;
    inboundTransactions_.newRound (previousLedger_->info().seq);
```